homeassistant_cli/config.py

```diff
@@ -4,6 +4,7 @@
 import click
 
 from homeassistant_cli.const import DEFAULT_HOST
+from homeassistant_cli.const import DEFAULT_PORT
```

Comment:
Updated, still learning python. Just wrapped my head around the different import methods now.